packages/lesmis/src/views/DashboardView.js

```diff
@@ -439,23 +479,30 @@ export const DashboardView = React.memo(() => {
           <VitalsView vitals={vitals} />
         </Container>
       </Wrapper>
-      {dropdownOptions.map(({ value, Body, Component }) => (
+      {dropdownOptions.map(({ value, Body, Component, useYearSelector, ComponentProps }) => (
         <TabPanel key={value} isSelected={value === selectedOption} Panel={React.Fragment}>
           <Component
             entityCode={entityCode}
+            year={useYearSelector && selectedYear}
```

Comment:
```suggestion
            year={selectedYear}
```
Un less there is a reason to not just pass through the year and let the component decide whether it is used or not?

Reply:
Yeah the component doesn't know whether it should use a year selector, because I lifted the year selector out of the component entirely